app/gui/controller/engine-protocol/src/language_server/types.rs

```diff
@@ -108,7 +108,7 @@ impl Path {
 // ====================
 
 /// Notification generated by the Language Server.
-#[derive(Clone, Debug, Deserialize, PartialEq, Serialize, IntoStaticStr, Eq)]
```

Comment:
Why do we need to remove `Eq`?

Reply:
Because `ExpressionUpdatePayload` contains a variant that has a `f64` field.